### PR TITLE
Add possibility to add datasources to 3D view

### PR DESCRIPTION
**Pull request by gjn**
This PR adds the possibility to add datasources (KML, GEOJson) to the Cesium rendered view.

The handling of the datasources themselves are the responsibility of the clients of ol3-cesium.

### CODE REVIEW

File: Cesium.externs.js [hunk 1]
```diff
@@ -2333,7 +2333,7 @@ Cesium.Scene.prototype.initializeFrame = function() {};
 
 /**
  */
-Cesium.Scene.prototype.render = function() {};
+Cesium.Scene.prototype.render = function(time) {};
```
Comment (gberaudo) [inline Cesium.externs.js]: Missing param jsdoc.

File: examples/kml.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  view: new ol.View({
+    center: ol.proj.transform([25, 20], 'EPSG:4326', 'EPSG:3857'),
+    zoom: 3
+  })
+});
+
+var ol3d = new olcs.OLCesium({map: ol2d});
+var scene = ol3d.getCesiumScene();
+var terrainProvider = new Cesium.CesiumTerrainProvider({
+    url : '//cesiumjs.org/stk-terrain/tilesets/world/tiles'
```
Comment (gberaudo) [inline examples/kml.js]: The terrain URL should be changed to `//assets.agi.com/stk-terrain/world`.

**Comment by gberaudo** [comment]
Would be a great addition when it passes travis.

**Comment by gjn** [comment]
@gberaudo Should be good now.

**Comment by gberaudo** [comment]
Though some KML shows up, I see 404 errors in the network console.
It is for resources like `https://api3.geo.admin.ch/signature/oef.jpg`.

Anyway this PR is great and I will merge it now. Thank you @gjn.

**Pull request merged**